src/RoundTabler/PerformanceResult.java

```diff
@@ -0,0 +1,48 @@
+package RoundTabler;
+
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+
+
+public class PerformanceResult{
+
+	public String TableName;
+	public String TableColumn;
+	public int RowsScanned;
+	public int RowsMatched;
+	public String MatchType;    //"PCIDSS or NACHA"
+	public LocalDateTime	ScanStarted;
+	public LocalDateTime	ScanFinished;
+	public Double			RowsPerSecond;
+
+	public PerformanceResult(){
+		super();
+
+	}
+
+	private void CalculatePerformance(){
+		long elapsed = ChronoUnit.SECONDS.between( ScanStarted, ScanFinished);
+		if ( elapsed != 0 ) { 
+			Double dRows = new Double(RowsScanned);
+			Double dElapsed = new Double(elapsed);
+			RowsPerSecond = dRows / dElapsed; 
+					
+		} else { RowsPerSecond = 0.0;}
+
+	}
+
+	
+	public String toString(){
```

Comment:
calling the fuction toString might cause some issues since toString is a native fuction to Java. not sure but worth a second look?

Reply:
@zachrussell12  That is by design. toString() is the method java call for retrieving the String representation of an object (in this case, a PerformanceResult)

Reply:
Okay! just wanted to make sure it wouldn't cause any issues.